WordPress/src/test/java/org/wordpress/android/ui/mysite/ScanAndBackupSourceTest.kt

```diff
@@ -123,4 +128,21 @@ class ScanAndBackupSourceTest : BaseUnitTest() {
 
         assertThat(result!!.scanAvailable).isTrue
     }
+
+    @Test
+    fun `when refresh is invoked, then data is refreshed`() = test {
+        whenever(selectedSiteRepository.getSelectedSite()).thenReturn(site)
+        whenever(site.siteId).thenReturn(siteRemoteId)
+        whenever(jetpackCapabilitiesUseCase.getJetpackPurchasedProducts(siteRemoteId)).thenReturn(
+                flow { emit(JetpackPurchasedProducts(scan = true, backup = true)) }
+        )
+
+        var result: JetpackCapabilities? = null
```

Comment:
Minor (🔍): The variable `result` is assigned but never accessed. Consider checking on that.

Reply:
Removed in a77a921